docs/tutorials/BuildMultiClusterApplication/BuildMultiClusterApplication.md

```diff
@@ -0,0 +1,524 @@
+# Tutorial: Build a Multi-Cluster Application seamlessly with ClusterResourcePropagation and Multi-Cluster Networking APIs
+
+This tutorial will guide you through the process of building a multi-cluster application using the ClusterResourcePropagation and Multi-Cluster networking APIs.
+
+Here are few key reasons why multi-cluster applications are valuable:
+- High Availability and Disaster Recovery 
+   - Deploying applications across multiple clusters (in different regions) ensures availability even if one cluster or region goes down due to network issues, hardware failures, or natural disasters.
+- Scalability
+   - Distributing the load across multiple clusters helps in scaling the application horizontally.
+- Fault Isolation
+  - By spreading workloads across multiple clusters, the failure of one cluster can be isolated, preventing it from impacting the entire application.

```

Comment:
I think it will be great to give a high level flow of the tutorial instead as those who are interested in the tutorial already have a desire to build a multi-cluster application.
i.e. we will start with an app on one cluster, setup ATM, setup override, ask CRP to propagate it to another cluster